lib/jeff/reply/input_status.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Jeff.Reply.InputStatus do
+  @moduledoc """
+  Input Status Report
+
+  OSDP v2.2 Specification Reference: 7.7
+  """
+
+  defstruct [:inputs]
+
+  @type state :: :active | :inactive
+
+  @type t :: %__MODULE__{
+          inputs: %{non_neg_integer() => state()}
+        }
+
+  @spec new(map()) :: t()
+  def new(inputs) do
+    %__MODULE__{
+      inputs: inputs
+    }
+  end
+
+  @spec decode(<<_::_*8>>) :: t()
+  def decode(data) do
+    for(<<input::8 <- data>>, do: input_status(input))
+    |> Enum.with_index(&{&2, &1})
+    |> Enum.into(%{})
+    |> new()
+  end
+
+  defp input_status(0x00), do: :inactive
+  defp input_status(_), do: :active
```

Comment:
```suggestion
  defp input_status(0x01), do: :active
  defp input_status(status), do: status
```

So the OSDP spec allows for input supervision, which I'm interpreting in the docs that if it is supervised, this byte might be custom to the manufacturer and could be 2..255. Maybe we allow just returning the byte value here if its not `0` or `1` while we decide a better way to allow custom status mapping for inputs?
![image](https://user-images.githubusercontent.com/11321326/204387825-b591e6ce-a49c-4d75-a875-cd4c5b8057ec.png)

Reply:
Makes sense to me. 👍